tests/test_load.py

```diff
@@ -89,12 +89,18 @@ def test_point_load_getter(
     ULS_DL.add_load(Single)  # ch
     example_bridge.add_load_case(ULS_DL)
     og.ops.wipe()
-
+    #
+    # assert example_bridge.load_case_list[0]["load_command"] == [
+    #     "ops.load(12, *[0, 0.6075807082987842, 0, 0.37389582049155967, 0, 0.34166943132701877])\n",
+    #     "ops.load(17, *[0, 1.4724192917012129, 0, 0.9061041795084391, 0, -0.613915767971676])\n",
+    #     "ops.load(18, *[0, 12.685458513118162, 0, -3.6244167180337583, 0, -5.289120462525217])\n",
+    #     "ops.load(13, *[0, 5.234541486881841, 0, -1.4955832819662394, 0, 2.943613562202012])\n",
+    # ]
     assert example_bridge.load_case_list[0]["load_command"] == [
-        "ops.load(12, *[0, 0.6075807082987842, 0, 0.37389582049155967, 0, 0.34166943132701877])\n",
-        "ops.load(17, *[0, 1.4724192917012129, 0, 0.9061041795084391, 0, -0.613915767971676])\n",
-        "ops.load(18, *[0, 12.685458513118162, 0, -3.6244167180337583, 0, -5.289120462525217])\n",
-        "ops.load(13, *[0, 5.234541486881841, 0, -1.4955832819662394, 0, 2.943613562202012])\n",
+        "ops.load(12, *[0, np.float64(0.6075807082987842), 0, np.float64(0.37389582049155967), 0, np.float64(0.34166943132701877)])\n",
+        "ops.load(17, *[0, np.float64(1.4724192917012129), 0, np.float64(0.9061041795084391), 0, np.float64(-0.613915767971676)])\n",
+        "ops.load(18, *[0, np.float64(12.685458513118162), 0, np.float64(-3.6244167180337583), 0, np.float64(-5.289120462525217)])\n",
+        "ops.load(13, *[0, np.float64(5.234541486881841), 0, np.float64(-1.4955832819662394), 0, np.float64(2.943613562202012)])\n",
```

Comment:
Does `ops.load` not just take a raw list? Do we really need to wrap every float with a `np.float64`?